webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -41,15 +41,16 @@ import { useSearchParams } from "react-router-dom";
 import { useStorageConfigs } from "../../../lib/hooks/storageConfig";
 import { getRepoStorageConfig } from "./utils";
 import {useDropzone} from "react-dropzone";
-import Container from "react-bootstrap/Container";
-import Row from "react-bootstrap/Row";
-import Col from "react-bootstrap/Col";
 import pMap from "p-map";
 
 const README_FILE_NAME = "README.md";
 const REPOSITORY_AGE_BEFORE_GC = 14;
 const MAX_PARALLEL_UPLOADS = 5;
 
+const isAbortedError = (error, controller) => {
+  return (error instanceof DOMException && error.name === 'AbortError') || controller.signal.aborted;
+};
```

Comment:
When branch protection is on, an error is thrown and this code throw exception while access undefined (controller.signal.aborted) and we get a blank page with exception in the console.